Dockerfile

```diff
@@ -180,6 +184,9 @@ COPY --from=php-ext-gd /usr/local/lib/php/extensions/no-debug-non-zts-20180731/g
 # PHP extension intl
 COPY --from=php-ext-intl /usr/local/etc/php/conf.d/docker-php-ext-intl.ini /usr/local/etc/php/conf.d/docker-php-ext-intl.ini
 COPY --from=php-ext-intl /usr/local/lib/php/extensions/no-debug-non-zts-20180731/intl.so /usr/local/lib/php/extensions/no-debug-non-zts-20180731/intl.so
+# PHP extension ldap
+COPY --from=php-ext-ldap /usr/local/etc/php/conf.d/docker-php-ext-ldap.ini /usr/local/etc/php/conf.d/docker-php-ext-ldap.ini
```

Comment:
We have to check how long the ldap extension build will took.
i have ordered the copy's from fastest to longest build time.
so the finished ones can be copied while the others still building (multitasking)

Reply:
@Schrolli91 I also added the time the builds took on my machine for later reference.